packages/client/ui/react-ui/src/components/hosted/CrossmintVerificationButton.tsx

```diff
@@ -0,0 +1,12 @@
+import { CSSProperties, MouseEvent } from "react";
+
+import { CrossmintVerificationButtonProps } from "@crossmint/client-sdk-base";
+
+export type CrossmintVerificationButtonReactProps = CrossmintVerificationButtonProps & {
```

Comment:
do we use CrossmintPaymentsButton or something? Why the Crossmint preffix?

Reply:
Yes, the name of our SDK is CrossmintPayButton

![Screenshot 2024-03-05 at 18 41 11](https://github.com/Crossmint/crossmint-sdk/assets/20989060/9fcafa0a-dcd9-4b0e-bf19-57ff12b70d6e)

I don't know why the Crossmint prefix. But some other products use it as well. Example with Persona:
![Screenshot 2024-03-05 at 18 41 54](https://github.com/Crossmint/crossmint-sdk/assets/20989060/a58b271f-ed85-4fc1-b812-65c4aab2968d)